packages/expo-cli/src/commands/start.js

```diff
@@ -72,11 +78,13 @@ async function action(projectDir, options) {
   if (!nonInteractive && !exp.isDetachexped) {
     await TerminalUI.startAsync(projectDir, startOpts);
   } else {
-    if (!exp.isDetached) {
-      log.newLine();
-      urlOpts.printQRCode(url);
+    if (!options.webOnly) {
```

Comment:
When web is enabled, do we print the URL of the site?

Reply:
we open the devtools with the link to open it. May be nice to have both though.

Reply:
I think it'd be useful to show the URL in the CLI too for people who don't have the web UI of Expo CLI enabled (they just use CLI) and also so we clarify which URLs are for which platforms, now that we have multiple URLs.